internal/replacements/replace_failed_process_groups.go

```diff
@@ -73,6 +76,12 @@ func ReplaceFailedProcessGroups(log logr.Logger, cluster *fdbv1beta2.FoundationD
 			continue
 		}
 
+		for _, targets := range containerProcessGroups {
```

Comment:
We can remove the if statement above since the normalise method will take care of the conversion.